internal/schemautil/userconfig/apiconvert/toapi_test.go

```diff
@@ -678,7 +683,8 @@ func TestToAPI(t *testing.T) {
 						map[string]interface{}{
 							"namespaces": []interface{}{
 								map[string]interface{}{
-									"retention": "48h",
+									"resolution": "30s",
+									"retention":  "48h",
```

Comment:
Why you change namespaces tests?

Reply:
because the behavior of the `ToAPI` changed, we had a wrong behavior before